packages/terra-action-header/tests/wdio/action-header-spec.js

```diff
@@ -0,0 +1,130 @@
+/* global browser, Terra */
+const viewports = Terra.viewports('tiny', 'medium', 'large');
+
+describe('ActionHeader', () => {
+  describe('Default', () => {
+    beforeEach(() => browser.url('/#/raw/tests/action-header/default-action-header'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty(
+      {
+        '--terra-action-header-background-color': '#551A8B',
+        '--terra-action-header-border-bottom': '1px solid #00FF00',
+        '--terra-action-header-border-left': '1px solid #00FF00',
+        '--terra-action-header-border-right': '1px solid #00FF00',
+        '--terra-action-header-border-top': '1px solid #00FF00',
+        '--terra-action-header-box-shadow': '0 1px 0 0 #FF8C00',
+        '--terra-action-header-font-color': '#4682B4',
+        '--terra-action-header-font-family': 'Roboto',
+        '--terra-action-header-font-size': '0.5rem',
+        '--terra-action-header-padding': '0.714rem',
+      },
+    );
+  });
+
+  describe('Back', () => {
+    beforeEach(() => browser.url('/#/raw/tests/action-header/back-action-header'));
+
+    Terra.should.beAccessible({ viewports });
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.themeEachCustomProperty(
+      {
+        '--terra-action-header-background-color': '#551A8B',
+        '--terra-action-header-border-bottom': '1px solid #00FF00',
+        '--terra-action-header-border-left': '1px solid #00FF00',
+        '--terra-action-header-border-right': '1px solid #00FF00',
+        '--terra-action-header-border-top': '1px solid #00FF00',
+        '--terra-action-header-box-shadow': '0 1px 0 0 #FF8C00',
+        '--terra-action-header-font-color': '#4682B4',
+        '--terra-action-header-font-family': 'Roboto',
+        '--terra-action-header-font-size': '0.5rem',
+        '--terra-action-header-padding': '0.714rem',
```

Comment:
how is this working? it looks like we're just setting the padding to the [default value](https://github.com/cerner/terra-core/blob/terra-action-header/packages/terra-action-header/src/ActionHeaderContainer.scss#L11)?

I thought we were supposed to be setting them to random values to see if they took, but I could be wrong

Reply:
Good catch. Updated [here](https://github.com/cerner/terra-core/commit/50aed7a03dfe93b7dbe24a445f1395e38303e52a).